daemons/controld/controld_te_utils.c

```diff
@@ -386,7 +386,7 @@ abort_transition_graph(int abort_priority, enum pcmk__graph_next abort_action,
         const xmlNode *search = NULL;
 
         for(search = reason; search; search = search->parent) {
-            if (pcmk__xe_is(search, XML_TAG_DIFF)) {
+            if (pcmk__xe_is(search, PCMK_XE_DIFF)) {
```

Comment:
With `PCMK__XE_DIFF_ADDED` we went with internal for the patch format. Changing your mind?

Reply:
Nope. Public for v2 since we made an API schema: https://github.com/ClusterLabs/pacemaker/pull/3303#discussion_r1440662724. `PCMK_XE_DIFF` is the root node of the v2 format.

`PCMK__XE_DIFF_{ADDED,REMOVED}` are v1-only. I never made a schema for that, partly because it's legacy and partly because I could never make sense of how it was structured even after days of refactoring it.